src/explore-education-statistics-common/src/modules/release/components/ReleaseDataAndFilesAccordion.module.scss

```diff
@@ -0,0 +1,26 @@
+@import '~govuk-frontend/govuk/base';
+
+.container {
+  margin-bottom: govuk-spacing(9);
+  margin-top: -1 * (govuk-spacing(6));
+
+  @include govuk-media-query($until: tablet) {
+    margin-top: 0;
+  }
+
+  @include govuk-media-query($from: tablet) {
+    :global(.govuk-accordion__section-header) {
+      border-top: 0;
+    }
+  }
+
+  :global(.dfe-flex) {
+    @include govuk-media-query($until: tablet) {
+      display: block;
+    }
+  }
```

Comment:
Rather than overriding `dfe-flex` like this, would suggest creating a responsive variant of it in `_flex.scss` and using that instead e.g.

```scss
.dfe-flex--tablet {
   @include govuk-media-query($until: tablet) {
      display: flex;
   }
}
```

Reply:
I've done it differently in the scss module instead. Code style wise I don't really understand how the mix of utility and module styles is meant to work, is it better to add a new utility class instead?

Reply:
Think it depends! I would say that the utility styles are mostly there for really simple cases where you only need a few styles and it would just be unnecessary to create a new module class e.g. centring stuff with flex.

The main problem with loads of little module classes doing the same thing is that they gradually bloat the final build with more and more CSS, hence it's a little more efficient to have the utils in these cases.

I wouldn't say there's a very clear cut-off point between deciding to put it in a CSS module or using utils, but I'd suggest that if you're sticking 3/4 or more utilities on something, refactor it into a module class. Even more so if it needs some media queries. We especially want to avoid applying media queries on top of util classes, like the previous code!

I think doing this as a module class is fine though, I was originally going to suggest this! :slightly_smiling_face: There is reasonably a use-case for responsive variants of utils as well, but I guess we might want to think about that a little more before jumping into it wholesale.